styles/MainNavigator.module.less

```
@@ -0,0 +1,10 @@
+.navbar {
```

```suggestion
.nav-box {
```
这个不是 NavBar 的根节点，而是 Nav 的，所以语义改得清晰一点。